src/pages/landing.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import Image from '../components/image'
+import Layout from '../components/layout'
+import Button from '@material-ui/core/Button';
+
+
+const LandingPage = () => (
+    <Layout>    
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous"></link>
+        <h1>Select your Game</h1>
+        <div style={{ maxWidth: '300px', marginBottom: '1.45rem' }}>
+            <Image />
+        </div>
+        <div class="game-selection">
```

Comment:
I think this should be `className` instead of `class`. Not sure why it is working fine (Maybe due to it being in a const file), but `class` is reserved in JSX for something else. Better to keep it consistent. 
```suggestion
        <div className="game-selection">
```